src/main/webapp/WEB-INF/tags/instructor/feedbackEdit/customFeedbackPathsData.tag

```diff
@@ -0,0 +1,7 @@
+<%@ attribute name="sessionCreatorData" required="true" %>
+<%@ attribute name="studentsData" required="true" %>
+<%@ attribute name="instructorsData" required="true" %>
+
+<div id="session-creator-data" data-session-creator="${sessionCreatorData}"></div>
+<div id="students-data" data-students="${studentsData}"></div>
+<div id="instructors-data" data-instructors="${instructorsData}"></div>
```

Comment:
_Note: Actually instructor and student data are already being passed to the preview session form; unfortunately, they are wrapped in inaccessible options tag =(_